downstream/modules/platform/ref-controller-analytics-reports.adoc

```diff
@@ -10,16 +10,17 @@ Currently available is a view-only version of the Automation Calculator utility
 
```

Comment:
Reports are no longer available directly from the UI. You might want to reword this to say that they are accessible from console.redhat.com and removing/commenting out the statement about analytics on-prem, or specifying that analytics provided on prem are: Automation Calculator, Host Metrics and Subscription Usage but I'm assuming there may be sections for those already and this is specific to reports? If so, maybe change the paragraph to something like this (fyi...some descriptions were taken from tooltips so modify as you see fit!):

Reports for data collected are available through console.redhat.com. 

{Analytics} currently available and accessible through the platform UI include the following: 

Automation Calculator is a view-only version of the Automation Calculator utility that shows a report that represents (possible) savings to the subscriber. 

Host Metrics is an analytics report collected for host data such as, when they were first automated, when they were most recently automated, how many times they were automated, and how many times each host has been deleted.

Subscription Usage reports the historical usage of your subscription. Subscription capacity and licenses consumed per month are displayed, with the ability to filter by the last year, two years, or three years.

Reply:
Updated to reflect the 2.5 environment.